uds/utilities/common_types.py

```
@@ -50,6 +50,21 @@ def validate_raw_byte(value: int) -> None:
 
+def validate_raw_2byte_value(value: int) -> None:
+    """
+    Validate whether provided value stores a raw 2-byte value.
+
+    :param value: Value to validate.
+
+    :raise TypeError: Value is not int type.
+    :raise ValueError: Value is out of byte range (0x00-0xFF).
+    """
+    if not isinstance(value, int):
+        raise TypeError(f"Provided value is not int type. Actual type: {type(value)}")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)